### ISSUE TITLE
Avoid XML-specific terminology "IDREF"

### ISSUE THREAD

**Issue by zcorpan** [w3c/aria-practices#1258]
See https://github.com/w3c/aria/issues/952 - these terms are used in aria-practices as well and should be updated.

* IDREF and #IDREF should be ID reference (or maybe `ID_reference` in markup examples)
* IDREFS and IDREF_LIST should be ID reference list (or maybe `ID_reference_list` in markup examples)

<details>
<summary>A grep on current master for `IDREF`</summary>

```
$ git grep "IDREF"
aria-practices-DeletedSectionsArchive.html:      <p>The <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> property can be used to label all visual objects, however it should be noted that (X)HTML provides a @<code>for</code> attribute on the <code><a href="http://www.w3.org/TR/1999/REC-html401-19991224/interact/forms.html#h-17.9">label</a></code> element which is used to label form controls. Use this attribute where possible and valid. Because the aria-labelledby attribute accepts multiple IDREFs, it is recommended that authors use aria-labelledby for labeling elements that require more than one label string.</p>
aria-practices-DeletedSectionsArchive.html:      <p>The <code class="property-reference">aria-describedby</code> property is not designed to reference descriptions on an external resource&#8212;since it is an IDREF, it must reference an element in the same DOM document. This is different from the HTML <code>longdesc</code> attribute, which accepts any URI. In general, the preferred pattern for WAI-ARIA applications is to include all relevant resources in the same DOM, But if you wish to reference an external resource with <code class="property-reference">aria-describedby</code>, you can reference a link that in turn references the external resource. This requires the user to follow two steps, first following the <code class="property-reference">aria-describedby</code> arc, then following the link, but does address the use case. The following example demonstrates this:</p>
aria-practices.html:                It has <code>aria-activedescendant="IDREF"</code> where IDREF is the ID of the element within the container that should be identified as active when the widget receives focus.
common/terms.html:    <dt><dfn>Valid IDREF</dfn></dt>
examples/accordion/accordion.html:          <th scope="row"><code>aria-labelledby="IDREF"</code></th>
examples/carousel/carousel-1.html:                  <code>aria-controls="IDREF"</code>
examples/checkbox/checkbox-2/checkbox-2.html:                <th scope="row"><code>aria-controls=&quot;[IDREFS]&quot;</code></th>
examples/combobox/aria1.0pattern/combobox-autocomplete-both.html:              <code>aria-owns="#IDREF"</code>
examples/combobox/aria1.0pattern/combobox-autocomplete-both.html:              <code>aria-activedescendant="IDREF"</code>
examples/combobox/aria1.0pattern/combobox-autocomplete-list.html:              <code>aria-owns="#IDREF"</code>
examples/combobox/aria1.0pattern/combobox-autocomplete-list.html:              <code>aria-activedescendant="IDREF"</code>
examples/combobox/aria1.0pattern/combobox-autocomplete-none.html:              <code>aria-owns="#IDREF"</code>
examples/combobox/aria1.0pattern/combobox-autocomplete-none.html:              <code>aria-activedescendant="IDREF"</code>
examples/combobox/aria1.1pattern/grid-combo.html:              <code>aria-owns=<q>IDREF</q></code>
examples/combobox/aria1.1pattern/grid-combo.html:              <code>aria-controls=<q>IDREF</q></code>
examples/combobox/aria1.1pattern/grid-combo.html:              <code>aria-activedescendant=<q>IDREF</q></code>
examples/combobox/aria1.1pattern/grid-combo.html:              <code>aria-labelledby=<q>IDREF</q></code>
examples/combobox/aria1.1pattern/listbox-combo.html:              <code>aria-owns=<q>IDREF</q></code>
examples/combobox/aria1.1pattern/listbox-combo.html:              <code>aria-controls=<q>IDREF</q></code>
examples/combobox/aria1.1pattern/listbox-combo.html:              <code>aria-activedescendant=<q>IDREF</q></code>
examples/combobox/aria1.1pattern/listbox-combo.html:              <code>aria-labelledby=<q>IDREF</q></code>
examples/dialog-modal/alertdialog.html:              <th scope="row"><code>aria-labelledby=<q>IDREF</q></code></th>
examples/dialog-modal/alertdialog.html:              <th scope="row"><code>aria-describedby=<q>IDREF</q></code></th>
examples/dialog-modal/datepicker-dialog.html:                <th scope="row"><code>aria-labelledby="IDREF"</code></th>
examples/dialog-modal/datepicker-dialog.html:              <th scope="row"><code>aria-labelledby=<q>IDREF</q></code></th>
examples/dialog-modal/dialog.html:            <th scope="row"><code>aria-labelledby=<q>IDREF</q></code></th>
examples/dialog-modal/dialog.html:            <th scope="row"><code>aria-describedby=<q>IDREF</q></code></th>
examples/disclosure/disclosure-faq.html:            <code>aria-controls=&quot;IDREF&quot;</code>
examples/disclosure/disclosure-faq.html:                The disclosure button controls visibility of the container identified by the <code>IDREF</code> value.
examples/disclosure/disclosure-img-long-description.html:            <code>aria-controls=&quot;IDREF&quot;</code>
examples/disclosure/disclosure-img-long-description.html:              <li>The disclosure button controls visibility of the container identified by the <code>IDREF</code> value.</li>
examples/disclosure/disclosure-navigation.html:            <code>aria-controls="IDREF"</code>
examples/disclosure/disclosure-navigation.html:            The disclosure button controls visibility of the container identified by the <code>IDREF</code> value.
examples/feed/feed.html:          <th scope="row"><code>aria-labelledby="IDREF"</code></th>
examples/feed/feed.html:          <th scope="row"><code>aria-labelledby="IDREF_LIST"</code></th>
examples/feed/feed.html:          <th scope="row"><code>aria-describedby="IDREF_LIST"</code></th>
examples/landmarks/js/visua11y.js:*   separated list of IDREFs), visit each referenced element in the order it
examples/menu-button/menu-button-actions-active-descendant.html:              <th scope="row"><code>aria-controls=&quot;IDREF&quot;</code></th>
examples/menu-button/menu-button-actions-active-descendant.html:              <th scope="row"><code>aria-labelledby=&quot;IDREF&quot;</code></th>
examples/menu-button/menu-button-actions-active-descendant.html:              <th scope="row"><code>aria-activedescendant=&quot;IDREF&quot;</code></th>
examples/menu-button/menu-button-actions-active-descendant.html:                  <li>The <code>IDREF</code> value is updated when keys that move the focus indicator among menu items are pressed.</li>
examples/menu-button/menu-button-actions.html:              <th scope="row"><code>aria-controls=&quot;IDREF&quot;</code></th>
examples/menu-button/menu-button-actions.html:              <th scope="row"><code>aria-labelledby=&quot;IDREF&quot;</code></th>
examples/menu-button/menu-button-links.html:              <th scope="row"><code>aria-controls=&quot;IDREF&quot;</code></th>
examples/menu-button/menu-button-links.html:              <th scope="row"><code>aria-labelledby=&quot;IDREF&quot;</code></th>
examples/radio/radio-1/radio-1.html:              <th scope="row"><code>aria-labelledby="[IDREF]"</code></th>
examples/radio/radio-2/radio-2.html:              <th scope="row"><code>aria-labelledby="[IDREF]"</code></th>
examples/radio/radio-2/radio-2.html:              <th scope="row"><code>aria-activedescendant=&quot;[IDREF]&quot;</code></th>
examples/slider/slider-1.html:              <code>aria-labelledby=<q>IDREF</q></code>
examples/slider/slider-2.html:              <code>aria-labelledby=<q>IDREF</q></code>
examples/spinbutton/datepicker-spinbuttons.html:            <th scope="row"><code>aria-labelledby="IDREFs"</code></th>
examples/table/table.html:            <code>aria-describedby=&quot;IDREF&quot;</code>
examples/tabs/tabs-1/tabs.html:                <code>aria-controls=<q>IDREF</q></code>
examples/tabs/tabs-1/tabs.html:                <code>aria-labelledby=<q>IDREF</q></code>
examples/tabs/tabs-2/tabs.html:                <code>aria-controls=<q>IDREF</q></code>
examples/tabs/tabs-2/tabs.html:                <code>aria-labelledby=<q>IDREF</q></code>
examples/toolbar/toolbar.html:              <code>aria-controls="IDREF"</code>
examples/toolbar/toolbar.html:            <th scope="row"><code>aria-controls="IDREF"</code></th>
examples/treeview/treeview-1/treeview-1a.html:              <th scope="row"><code>aria-labelledby=&quot;IDREF&quot;</code></th>
examples/treeview/treeview-1/treeview-1b.html:              <th scope="row"><code>aria-labelledby=&quot;IDREF&quot;</code></th>
examples/treeview/treeview-2/treeview-2a.html:              <th scope="row"><code>aria-labelledby=&quot;IDREF&quot;</code></th>
examples/treeview/treeview-2/treeview-2b.html:              <th scope="row"><code>aria-labelledby=&quot;IDREF&quot;</code></th>
```

</details>
